src/remote_connector_dao/RealTimeGraphsWidget.py

```diff
@@ -0,0 +1,116 @@
+from PyQt5 import QtCore
+from PyQt5.QtWidgets import QVBoxLayout, QWidget
+import pyqtgraph as pg
+from remote_connector_dao.SignalCounter import SignalCounter
+from remote_connector_dao.get_signal import get_count_rates
+from remote_connector_dao.constants import GRAPH_ANIMATION_INTERVAL
+from remote_connector_dao.app_styles import *
+
+
+class RealTimeGraphsWidget(QWidget):
+    def __init__(
+        self,
+        timetagger_proxy,
+        timetagger_controller,
+        channels,
+    ):
+        super().__init__()
+        self.signal_counter = SignalCounter()
+        self.channels_list = channels
+        self.timetagger_proxy = timetagger_proxy
+        self.timetagger_controller = timetagger_controller
+
+        self.elapsed_time = self.signal_counter.elapsed_time
+        self.channel1_counts = self.signal_counter.channel1
+        self.channel2_counts = self.signal_counter.channel2
+        self.coincidences = self.signal_counter.coincidences
+        self.single_counts_max_value = self.signal_counter.single_channels_max_value
```

Comment:
é necessário criar esse nível de indireção aqui? 

Além de prejudicar a legibilidade, isso pode ser bem perigoso:
- isso não funcionaria pra tipos primitivos, pois seria feita uma copia do valor naquele momento
- para referências, como é o caso desses atributos, alguma parte dessa classe poderia acabar alterando uma dessas referências sem saber que ela pertence a outro objeto. Ai vc choraria bastante ate achar o erro haahha

É melhor vc referenciar os atributos através do objeto `signal_counter`.
**Se for muitooo necessário** expor esses atributos como parte da classe RealTimeGraphsWidget, é melhor vc utilizar uma property, que aí deixa claro a relação

```
@property
def channel1_counts(self):
    return self.signal_counter.channel1
```

Reply:
Não é necessário criar essa indireção. Vou remover isso aqui, posso usar direto os atributos da `SignalCounter`